api/app/signals/apps/api/v1/views/signal.py

```diff
@@ -49,6 +50,62 @@ def retrieve(self, request, signal_id):
         return Response(data)
 
 
+def _get_list_json():
```

Comment:
Please make this part of the view instead of a function OR move it to a utils file. It is a bit out of place in a class based view setup :)

Reply:
moved into view, since it's only used here. On next usage it will be moved to utils.